docs/API-V1/3-FILTER-PORTS.md

```diff
@@ -0,0 +1,19 @@
+## Filter Ports
+
+#### `- GET /ports/`
```

Comment:
I miss the part that just gives you the count of ports, and a way to limit the results that you fetch. (If one would want to create some UI with pagination, it would be nice to just get the info for the 20 desired ports.)